mail-client/neomutt-test-files/Manifest

```diff
@@ -0,0 +1 @@
+DIST neomutt-test-files-1.tar.gz 3673 BLAKE2B 50354f19aedc5fc07d59ecb5b38fa65de16119bc0929f47014bd35dab4d3a4e5953c8b35670f3440003cf0cbccc2d0d2b3a869dc929cdc3cd60b02790270fb54 SHA512 3d4962210bc558234d818801dcaa7851a0aef011c96d91c054af535186ffda42059fc61fb148c48e1b076999fe3159b31589a69a29ed1897f8928f52fcc157d3
```

Comment:
We have rules how to name packages such as this, 
https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#snapshots-and-live-ebuilds

`neomutt-test-files-0_pre20200618.ebuild`
So when -1.0 is released, it's going to be higher than the snapshot version.

Reply:
Fixed